test/fn/watch-project.spec.js

```diff
@@ -152,7 +152,7 @@ describe('watch-project', () => {
       .then(mockApi.getAppSettings)
       .then((settings) => JSON.parse(settings.content))
       .then((settings) => expect(settings.locale).equal('es'));
-  });
+  }).timeout(16000);
```

Comment:
I removed the 3 timeouts you added in this PR and ran `npm run test` 20+ times back to back with 0 failures. Do we really need them?

Reply:
:thinking: Hmm, thanks for the double-check here. This seems to confirm the problem is on my end (and not with the tests themselves).  I will go ahead and remove them.